hugo/data/home/home.yml

```diff
@@ -1,37 +1,39 @@
-hero:
-  title: Building the technological stack for AI on code
-  desc: source{d} is building the open-source components that enable machine learning on source code. We believe that the next step in programming comes from teaching machines to understand and write code.
+title: "building the technological stack for AI on code"
```

Comment:
I would like to have the sentence here capitalized so we make sure this is done in the stylesheet.